src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketBuildStatusClientImpl.java

```diff
@@ -29,6 +27,6 @@ public void post(BitbucketBuildStatus buildStatus) {
                 .addPathSegment("commits")
                 .addPathSegment(revisionSha)
                 .build();
-        bitbucketRequestExecutor.makePostRequest(url, buildStatus, Headers.of(Collections.emptyMap()));
+        bitbucketRequestExecutor.makePostRequest(url, buildStatus, new RetryOnRateLimitConfig(3));
```

Comment:
Is there a lightweight way we can configure this? I don't think the UI is the right place to put it because it seems unlikely that someone would want to change it, but I think that making it configurable would be best if possible.